### ISSUE TITLE
[Request] Update debian package to 22.01.6

### ISSUE THREAD

**Issue by susurri** [Mercury-Language/packaging#14]
Debian package is 22.01, which is several releases behind the current release 22.01.6.

**Comment by PaulBone** [comment]
Yes.  Thanks for the reminder.  Unfortunately I haven't had time to do this yet.  Hopefully soon, along with aarch64 builds.
